auth-bearer-jwt/Sources/App/Application+configure.swift

```diff
@@ -0,0 +1,24 @@
+import Hummingbird
+import HummingbirdAuth
+import HummingbirdFoundation
+
+extension HBApplication {
+  public func configure() throws {
+    let env = HBEnvironment()
```

Comment:
Thanks!

Reply:
If you add a comment header prefixed with a '///' it will automatically get added to the reference documentation the next time I build it. If you want to write more user-guide documentation then you can add markdown files to https://github.com/hummingbird-project/hummingbird/tree/main/documentation. Again they will automatically get added to the documentation when it is rebuilt